### DISCUSSION TITLE
word in different situation has different lemma_

### DISCUSSION THREAD

**Discussion by JJplane** [explosion/spaCy#6612] [Help: Other Questions]
```
import spacy
nlp = spacy.load('en_core_web_sm', disable=['ner', 'parser', 'textcat'])
nlp.add_pipe(nlp.create_pipe('sentencizer'))
doc = nlp('womaned_made'.replace("_"," "))
print(doc[0].lemma_)
doc1 = nlp('womaned'.replace("_"," "))
print(doc1[0].lemma_)
```
the output is:
```
womaned
womane
```
so why the same word has different lemma?
* Operating System:windows 10
* Python Version Used:3.6
* spaCy Version Used:2.3.5
* Environment Information:

**Comment by svlandeg** [comment]
FYI - we converted this question to a thread on the discussion board.

We're aiming to keep the Issue tracker focused mainly on bug reports, while the new GH Discussion board can be a place where the community can come together, help eachother, discuss best practices, and so on.
